pkg/cmd/config.go

```diff
@@ -90,6 +96,20 @@ func setConfigValue(key, value string) error {
 	return saveConfig()
 }
 
+func listAllConfig(out io.Writer) error {
+	config, err := unmarshalConfig()
+	if err != nil {
+		return nil
+	}
+
+	// Iterating over structs is hard and there are only two configuration
+	// directives at time of writing. Hard-code the output format for now, and
+	// invest in a more maintainable method in the future.
+	_, err = fmt.Fprintf(out, "user.name=%s\nuser.email=%s\n",
```

Comment:
That's a good point. You could get around the unknown directives by checking if the keys are valid, but that still leaves the nested keys. I'm just feeling a bit nervous about the directives being repeated so many times :) It's manageable now, but I would like this to be refactored before any other directives get added.